docs/source/developer/bevel_minikube_setup.md

```diff
@@ -9,6 +9,18 @@
 
 Before proceeding, first make sure that you've completed [Developer Pre-requisites](./dev_prereq.md).
 
+## Setup Minikube
+
+1. Install Minikube https://minikube.sigs.k8s.io/docs/start/ on VM
+
+2. Start minikube. The setup has been tested on a Ubuntu 20.4 VM which has a public ip
```

Comment:
Typo `20.04 VM`

Reply:
I have created new file for fabric deployments on minikube. kindly review